96-chromebook.install

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/bash
+
+COMMAND="$1"
+KERNEL_VERSION="$2"
+BOOT_DIR_ABS="$3"
+KERNEL_IMAGE="$4"
+
+if [[ -d "$BOOT_DIR_ABS" ]]; then
+    INITRD="initrd"
+else
+    BOOT_DIR_ABS="/boot"
+    INITRD="initramfs-${KERNEL_VERSION}.img"
+fi
+
+case "$COMMAND" in
+    add)
+        echo "Install required packages"
+        dnf install lz4 vboot-utils uboot-tools -y
```

Comment:
I would prefer if this script instead was shipped let's say as a part of vboot-utils? or maybe a vboot-utils-kernel-install sub-package and pulled these as dependencies ?

Reply:
just to make it clear, I don't think that the script should install packages. In any case if you want to ship this with `chromebook-setup.sh`, then these packages should be added as dependencies in the script itself using `ensure_command`.

Reply:
like do you mean submitting a patch to include the 96-chromebook.install script in vboot-utils ustream?

Reply:
yes. But I would ship it as a separate sub-package since it may be that someone wants to have the vboot-utils tools in their host machine for cross-compile and then ship to a target device. So a `vboot-kernel-install` or something sub-package would make sense.

That way you won't need to manually install the dependencies for this since it would be handled by the package manager.

Reply:
oh I get what you mean now, thanks.